### PR TITLE
chore: Upgrades ES to 7.13.3

### PR DESCRIPTION
**Pull request by verdan**
Signed-off-by: verdan <verdan.mahmood@gmail.com>

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Comment by feng-tao** [comment]
that's cool. I recall the mapping field is no longer needed in 7.x. Does the change intent to support both ES 6.x and 7.x?

**Comment by verdan** [comment]
> that's cool. I recall the mapping field is no longer needed in 7.x. Does the change intent to support both ES 6.x and 7.x?

Yes, the intent is to support both 6.x and 7.x.

**Comment by mgorsk1** [comment]
we still have some mapping definitions in amundsen common that need to be refactored

File: databuilder/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright Contributors to the Amundsen project.
 # SPDX-License-Identifier: Apache-2.0
 
-elasticsearch>=6.2.0,<7.0
+elasticsearch>=6.2.0,<7.13.4
```
[the remark this answers is not in the document]
Comment (verdan) [reply databuilder/requirements.txt]: 👍

File: search/search_service/proxy/elasticsearch.py [hunk 2]
```diff
@@ -311,6 +318,9 @@ def _search_helper(self, page_index: int,
         :param query_name: name of query to query the ES
         :return:
         """
+        # This is to support ESv7.x
+        # ref: https://www.elastic.co/guide/en/elasticsearch/reference/7.0/breaking-changes-7.0.html#track-total-hits-10000-default # noqa: E501
+        client = client.extra(track_total_hits=True)
```
[the remark this answers is not in the document]
Comment (verdan) [reply search/search_service/proxy/elasticsearch.py:323]: Nope. working just fine.

**Review by mgorsk1 — APPROVED** [review]
lgtm

**Comment by mgorsk1** [comment]
I would disregard #1640 and go straight to es 8.0.0 which is used here https://github.com/amundsen-io/amundsen/blob/main/docker-amundsen.yml

**Pull request merged**